cfme/services/service_catalogs/ui.py

```diff
@@ -128,16 +128,9 @@ def order(self):
         view.fill(self.dialog_values)
     if self.ansible_dialog_values:
         view.fill(self.ansible_dialog_values)
-    if self.appliance.version < "5.9":
-        msg = "Order Request was Submitted"
-        msg_type = "success"
-    else:
-        msg = "Dialog submitted successfully!"
-        msg_type = "info"
```

Comment:
IMHO - we don't need this at all. it just gives no value. we have only these tests which are testing text from flash messages - https://github.com/lkhomenk/integration_tests/blob/d54791a439584da3a912b483e6bb417f4c60f84d/cfme/tests/cloud/test_providers.py#L33. other than that - we just need to order service. assert_no_error is enough

Reply:
@lkhomenk : this message in 5.9 "msg = "Dialog submitted successfully!" is a bug and should go in next build may be .https://bugzilla.redhat.com/show_bug.cgi?id=1518430

Reply:
@sshveta yes. that's true. but such cases should be separate tests like in the link I've pasted - separate test to check if flash message is correct. otherwise all other test  provide no value/metrics if they are failed on this - from jenkins side it will be failed "test_pxe_servicecatalog". I won't even put soft_assert there - all assertions supposed to be inside test, but not in helper functions etc.